go/vt/tabletserver/schema_info.go

```diff
@@ -409,6 +410,10 @@ func (si *SchemaInfo) DropTable(tableName string) {
 
 // GetPlan returns the ExecPlan that for the query. Plans are cached in a cache.LRUCache.
 func (si *SchemaInfo) GetPlan(ctx context.Context, logStats *LogStats, sql string) *ExecPlan {
+	span := trace.NewSpanFromContext(ctx)
+	span.StartLocal("SchemaInfo.GetPlan")
+	defer span.Finish()
+
```

Comment:
This will likely not be substantial, unless the client is sending non-bindvar queries all the time.